store/es/index.js

```diff
@@ -156,11 +156,13 @@ export default {
     createString: [{
         name: 'toString',
         shortDesc: `cadena de caracteres representando al objeto`,
-        desc: `El método <code>toString()</code> devuelve una cadena de caracteres que representa el objeto.`,
+        desc: `El método <code>toString()</code> devuelve una cadena de caracteres que representa el objeto. Aunque este es el método incorporado disponible, normalmente es más útil usar <a href="https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/JSON/stringify" target="_blank"><code>JSON.stringify()</code></a>.`,
         example: `console.log(obj.toString());<br>
-        console.log(obj.a.toString());`,
+        console.log(obj.a.toString());<br>
+ +      console.log(JSON.stringify(obj));`,
```

Comment:
This isn't in the original, it's just a note- I'd like to keep it this way so that we are strictly showing the one method and keep it from being confusing.

Reply:
Am I correcting the change? @sdras

Reply:
Changes made.